packages/client/tests/functional/interactive-transactions/tests.ts

```diff
@@ -1,15 +1,20 @@
 import { ClientEngineType, getClientEngineType } from '@prisma/sdk'
-import path from 'path'
 
-import { getTestClient } from '../../../../utils/getTestClient'
-import { tearDownPostgres } from '../../../../utils/setupPostgres'
-import { migrateDb } from '../../__helpers__/migrateDb'
+import testMatrix from './_matrix'
 
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+// @ts-ignore this is just for type checks
+declare let PrismaClient: typeof import('@prisma/client').PrismaClient
+
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
 const testIf = (condition: boolean) => (condition ? test : test.skip)
 
-let PrismaClient, prisma
+testMatrix.setupTestSuite(({ provider }) => {
+  beforeEach(async () => {
+    await prisma.user.deleteMany()
```

Comment:
After each test, we drop the DB https://github.com/prisma/prisma/blob/main/packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts#L62, and foreach test we setup a new db, does this need to be done?

Reply:
Would using unique values help, such as using something like faker?

Reply:
I think it might, let me try that

Reply:
Thanks for checking!